content/en/docs/tasks/configure-pod-container/user-namespaces.md

```diff
@@ -0,0 +1,214 @@
+that the last value is 4294967295, this means user namespaces are not active.
+
+The value 4294967295 is the max number of UIDs/GIDs a Linux system can use. If
+you see the whole range of UIDs/GIDs available to the pod, it means a user
+namespaces is not being used.
+
+If your pod's containers doesn't have a shell, you can do it from the host.
+
+If you know which PID in the host the container process is using, you can read
+from the host the same files: `cat /proc/$pid/uid_map` and `cat
+/proc/$pid/gid_map`, replacing `$pid` with the container PID.
+
+There are others ways that can be used to check if user namespaces are active or
+not. This section is not intended to list them all.
+
+## Limitations
+
+When using a user namespace for the pod, it is disallowed to use other host
+namespaces. In particular, if `hostUsers: false` then it is not allowed to set
+either of:
+
+ * `hostNetwork: true`
+ * `hostIPC: true`
+ * `hostPID: true`
+
+The pod is allowed to use no volumes at all or, if using volumes, only these
+volume types are allowed:
+
+ * configmap
+ * secret
+ * projected
+ * downwardAPI
+ * emptyDir
+
+To guarantee that the pod can read the files of such volumes, volumes are
+created as if `pod.spec.securityContext.FSGroup` is specified as `0`. If it is
+specified to a different value, this other value will of course be honored
+instead.
+
+As a by-product of this, folders and files for these volumes will have
+permissions for the group, even if defaultMode or mode to specific items of the
+volumes were specified without permissions to groups. For example, it is not
+possible to mount these volumes in a way that its files have permissions only
+for the owner. This is a limitation imposed by FSGroup. 
+
+This is a limitation in Kubernetes v1.25 and future versions of Kubernetes might
+remove this limitation when user namespaces are in use.
+
+## Recommended node set-up
+
+It is recommended that the host's files and host's process use UIDs/GIDs in the
+range of 0-65535.
+
+The kubelet will assign UIDs/GIDs higher than that to pods. Therefore, to
+guarantee as much isolation as possible, the UIDs/GIDs used by the host's files
+and host's processes should be in the range 0-65535.
+
+Note that this recommendation is important to mitigate the impact of CVEs like
+[CVE-2021-25741][CVE-2021-25741], where a pod can potentially read arbitrary
+files in the hosts. If the UIDs/GIDs of the pod and the host don't overlap, it
+is limited what a pod would be able to do: the pod UID/GID won't match the
+host's file owner/group.
+
+[CVE-2021-25741]: https://github.com/kubernetes/kubernetes/issues/104980
+
+## Important considerations
+
+The kubelet will decide automatically to which host UIDs/GIDs a pod will be
+mapped to, and will do so in a way to guarantee that no two stateless pods on
+the same node use the same mapping. 
+
+The `RunAsUser`, `RunAsGroup`, `FSGroup`, etc. fields in the `pod.spec` always
+refer to the user inside the container.
```

Comment:
Please write these fields with the capitalization that they have in the HTTP API.

Reply:
Fixed